test/terraform/gke/outputs.tf

```diff
@@ -7,5 +7,9 @@ output "cluster_names" {
 }
 
 output "context_names" {
-  value = [for cl in google_container_cluster.cluster : format("gke_%s_%s_%s", var.project, var.zone, cl.name) ]
+  value = [for cl in google_container_cluster.cluster : format("gke_%s_%s_%s", var.project, var.zone, cl.name)]
+}
+
+output "kubeconfigs" {
```

Comment:
Can you add a note that if this output changes, also update .circleci/config.yml?

Reply:
This actually is a bad thing, so I've modified the circleci config to not have this implicit coupling and use this output directly instead.